changelog/6052.breaking.rst

```diff
@@ -1 +1 @@
-The `.JSOCClient` and every `.GenericClient` was passing all ``**kwargs`` to `parfive.Downloader.enqueue_file`, this was unintended and has been removed.
+The `.JSOCClient` and every `sunpy.net.dataretriever.GenericClient` was passing all ``**kwargs`` to `parfive.Downloader.enqueue_file`, this was unintended and has been removed.
```

Comment:
Ah, so that didn't link the first time.

Reply:
It was throwing warnings, because of the ambiguous references.